src/mame/swtpc/swtpc09.cpp

```diff
@@ -68,23 +68,89 @@
  F100 - ????  CDS disk 0 controller
  F300 - ????  CDS disk 1 controller
 
-The supplied flex and uniflex floppies all worked when this driver was
-created (2014-03-25), but have been broken since sometime in 2015.
-
-The message is the same for all 3 disks: Unable to identify the image format.
-
-Official notes:
-mess swtpc09 -flop1 DMAF2_BOOT.dsk
-press D to boot flex operating system. dir 0 will perform a directory of drive 0.
-
-mess swtpc09u -flop1 uflxbtsd.dsk
-type D to boot uniflex. This is a multiuser operating system, I need to add an extra terminal to this
-and then it would be cool to get it running multiuser.
-
-mess swtpc09d3 -window -flop1 UOS_3S_BOOT.dsk
-type D to boot uniflex. This one has potential to get harddisk support for uniflex in the future.
-
-mess swtpc09i  not working
+** Notes on the use with common disk images **
+
+The MAME emulates the floppy disk controller more thoroughly than some other
+common emulators for these systems. Other common emulators can ignore the disk
+density and disk side driven by the software. FLEX numbers sectors
+contiguously from one side to the other so a simple emulator need only map the
+track and sector numbers. Real disk images often mix single and double density
+formats, e.g. single density for the first track, and the software may not
+have been updated to correctly drive the side and density outputs, so disks
+for these other emulators might not work in the MAME. These disk images
+and their softare need to be updated - this is not working around limitations
+of the MAME, these would not be expected to work on real hardware either.
```

Comment:
You’re still using MAME as though it’s a generic term – “might not work in the MAME”, “limitations of the MAME”.

Reply:
The use seems typical of uses in the docs/ but happy to just not mention 'MAME' here.

Reply:
No it's not.

Reply:
@MooglyGuy I had already avoided the use of 'MAME', so that was all unnecessary.